### PR TITLE
Add provideDirectoriesForThemeBundle in AssetsProvider

### PR DESCRIPTION
**Pull request by oallain**
This PR is based on #98 she upgrade tests, because it is the right solution, I think.

### CODE REVIEW

**Comment by oallain** [comment]
Hello @Sylius/core-team can you approve running workflows ?

File: src/Asset/Installer/AssetsProvider.php [hunk 1]
```diff
@@ -48,7 +52,11 @@ public function provideDirectoriesForTheme(ThemeInterface $rootTheme): iterable
     public function provideDirectoriesForBundle(BundleInterface $bundle): iterable
     {
         yield $bundle->getPath() . '/Resources/public' => '/bundles/' . $this->getPublicBundleName($bundle);
-        yield $bundle->getPath() . '/public' => '/bundles/' . $this->getPublicBundleName($bundle);
+    }
+
+    public function provideDirectoriesForThemeBundle(BundleInterface $bundle): iterable
+    {
+        yield $bundle->getPath() . '/Resources/public' => '/bundles/_theme/' . $this->getPublicBundleName($bundle);
```
Comment (pamil) [inline src/Asset/Installer/AssetsProvider.php:59]:
```suggestion
        yield $bundle->getPath() . '/Resources/public' => '/bundles/_themes/' . $this->getPublicBundleName($bundle);
```

To be compatible with PathResolver implementation: `return sprintf('%s/_themes/%s/%s', $prefixPath, $theme->getName(), $relativePath);`

**Comment by pamil** [comment]
Does this PR solve the problem for your usecase?

**Comment by Roshyo** [comment]
For what it's worth, I confirm that it works on my environment.

**Comment by oallain** [comment]
Not really, i need to work on it asap.

**Comment by oallain** [comment]
After somes tests, i close this PR.

**Comment by maximehuran** [comment]
Hi @oallain 👋 

Why this PR is closed ? We still have the issue on #91

**Pull request closed without merging**